src/app/components/shared/demo-page-properties.component.html

```diff
@@ -1,6 +1,12 @@
 <sky-demo-page-content [sectionHeading]="sectionHeading">
   <ng-content></ng-content>
-  <ul>
-    <ng-content select="sky-demo-page-property"></ng-content>
-  </ul>
-</sky-demo-page-content>
+  <div class="property-table" role="table">
+			<div class="property-table-row property-table-header" role="row">
+				<div class="property-name" role="columnheader">Property name</div>
```

Comment:
Does this header work for all scenarios? For example, does "property name" work for the tables with lists of events? i think it's probably fine, but I just wanted to raise the question to make sure.

Reply:
Good question, I'm not sure. Devs can probably weigh in on that.

Reply:
I'll bring this up at the weekly check-in meeting.